README.md

```diff
@@ -19,6 +19,7 @@
 | `rails importer:initial_featured` | sets up the "original" featured practices to show up on the landing page - depends on spreadsheet being imported | 
 | `rails surveymonkey:download_response_files` (DEPRECATED) | Rake task to download files from our SurveyMonkey practice submission form. **Do not use this anymore. Ever.**  | 
 | `rails diffusion_history:all`| Imports all of the diffusion history we have so far for practices - used to populate the geolocation feature (Practice <-> Facility mappings) |
+| `rails milestones:milestone_transfer` | Transfers all of the original timeline entry milestones to the new milestone model  |
```

Comment:
this is suppose to be 'milestones:milestones_transfer' (the extra 's' in the second milestone)

Reply:
Updated!